internalshared/configutil/config.go

```diff
@@ -13,6 +14,10 @@ import (
 
 // SharedConfig contains some shared values
 type SharedConfig struct {
+	FoundKeys  []string               `hcl:",decodedFields"`
+	UnusedKeys map[string][]token.Pos `hcl:",unusedKeyPositions"`
```

Comment:
did you want this to be of type `UnusedKeyMap` as defined in `config_util.go`?

Reply:
Ah nice, I must have added that type later but didn't loop back.